tensorflow/lite/experimental/swift/BUILD.apple

```diff
@@ -10,6 +10,16 @@ package(
     licenses = ["notice"],  # Apache 2.0
 )
 
+config_setting(
+    name = "use_coreml_delegate",
+    values = {"define": "use_coreml_delegate=1"},
+)
+
+config_setting(
+    name = "use_metal_delegate",
+    values = {"define": "use_metal_delegate=1"},
+)
+
 # TODO(b/153554551): investigate if separate delegate libraries can be made with same module_name
```

Comment:
I think we can remove this TODO to avoid confusion as your PR is addressing the issue.

Reply:
Done!